manager/integration/tests/test_snapshot.py

```diff
@@ -300,8 +300,17 @@ def prepare_settings_for_snapshot_test(client, data_integrity, immediate_check,
 
     cronjob = f"{minutes} {hours} * * *"
 
+    if DATA_ENGINE == "v1":
+        snapshot_data_integrity_setting = SETTING_SNAPSHOT_DATA_INTEGRITY
+        snapshot_fast_data_rebuild_enabled_setting = \
+            SETTING_SNAPSHOT_FAST_REPLICA_REBUILD_ENABLED
+    elif DATA_ENGINE == "v2":
```

Comment:
Why was `v2` related code flow added, but v2 support annotation `@pytest.mark.v2_volume_test` removed?

Reply:
Please see the [Special notes for your reviewer](https://github.com/longhorn/longhorn-tests/pull/2484#issue-3055801375), thank you.

Reply:
> Please see the [Special notes for your reviewer](https://github.com/longhorn/longhorn-tests/pull/2484#issue-3055801375)


According to above, `SNAPSHOT_DATA_INTEGRITY_ENABLED`(`enabled`) did not support on global setting `v2-data-engine-snapshot-data-integrity`, so related pytest mark removed because test case will fail in below scenario when test ing v2 data enagine
```
prepare_settings_for_snapshot_test(client,
                                       SNAPSHOT_DATA_INTEGRITY_ENABLED,
                                       "true", "true")
```